jpo-geojsonconverter/src/main/java/us/dot/its/jpo/geojsonconverter/converter/map/MapProcessedJsonConverter.java

```diff
@@ -1,401 +1,429 @@
-// offsetY = nodexy.getY().intValue();
-// }
-// else if (nodeLatLong != null) {
-// offsetX = nodeLatLong.getLon().divide(new BigDecimal("10000000")).intValue();
-// offsetY = nodeLatLong.getLat().divide(new BigDecimal("10000000")).intValue();
-// }
-// Integer[] delta = {offsetX,offsetY};
-// mapNode.setDelta(delta);
-// mapNodes.add(mapNode);
-// }
-// } catch (Exception e) {
-// String errMsg = String.format("Failed to nodeConversionList - SpatProcessedJsonConverter. Message: %s",
-// e.getMessage());
-// logger.error(errMsg, e);
-// }
-// return mapNodes;
-// }
-
-// }
+package us.dot.its.jpo.geojsonconverter.converter.map;
+
+import com.networknt.schema.ValidationMessage;
+import java.time.Instant;
+import java.time.ZoneId;
+import java.time.ZoneOffset;
+import java.time.ZonedDateTime;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.List;
+import org.apache.commons.lang3.exception.ExceptionUtils;
+import org.apache.kafka.streams.KeyValue;
+import org.apache.kafka.streams.kstream.Transformer;
+import org.apache.kafka.streams.processor.ProcessorContext;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import us.dot.its.jpo.asn.j2735.r2024.Common.MinuteOfTheYear;
+import us.dot.its.jpo.asn.j2735.r2024.Common.NodeOffsetPointXY;
+import us.dot.its.jpo.asn.j2735.r2024.Common.NodeSetXY;
+import us.dot.its.jpo.asn.j2735.r2024.Common.NodeXY;
+import us.dot.its.jpo.asn.j2735.r2024.Common.Node_LLmD_64b;
+import us.dot.its.jpo.asn.j2735.r2024.MapData.Connection;
+import us.dot.its.jpo.asn.j2735.r2024.MapData.GenericLane;
+import us.dot.its.jpo.asn.j2735.r2024.MapData.IntersectionGeometry;
+import us.dot.its.jpo.asn.j2735.r2024.MapData.MapData;
+import us.dot.its.jpo.asn.j2735.r2024.MapData.MapDataMessageFrame;
+import us.dot.its.jpo.geojsonconverter.partitioner.RsuIntersectionKey;
+import us.dot.its.jpo.geojsonconverter.pojos.ProcessedValidationMessage;
+import us.dot.its.jpo.geojsonconverter.pojos.geojson.LineString;
+import us.dot.its.jpo.geojsonconverter.pojos.geojson.connectinglanes.*;
+import us.dot.its.jpo.geojsonconverter.pojos.geojson.map.*;
+import us.dot.its.jpo.geojsonconverter.utils.ProcessedSchemaVersions;
+import us.dot.its.jpo.geojsonconverter.validator.CTI4501Validator;
+import us.dot.its.jpo.geojsonconverter.validator.JsonValidatorResult;
+import us.dot.its.jpo.ode.model.OdeMessageFrameData;
+import us.dot.its.jpo.ode.model.OdeMessageFrameMetadata;
+
+public class MapProcessedJsonConverter
+        implements Transformer<Void, DeserializedRawMap, KeyValue<RsuIntersectionKey, ProcessedMap<LineString>>> {
+    private static final Logger logger = LoggerFactory.getLogger(MapProcessedJsonConverter.class);
+
+    @Override
+    public void init(ProcessorContext arg0) {}
+
+    /**
+     * Transform an ODE MAP POJO to MAP GeoJSON POJO.
+     *
+     * @param rawKey - Void type because ODE topics have no specified key
+     * @param rawValue - The raw POJO
```

Comment:
nitpick: javadoc param name validation
```suggestion
     * @param rawMap - The raw POJO
```